Makefile

```diff
@@ -351,3 +351,26 @@ install: _output/local/bin
           done ; \
         fi
 endif
+
+define RELEASE_HELP_INFO
+# release components.
+#
+# Args:
+#   WHAT: Component names to be release. support: kubeedge/edgesite/keadm
+#         If not specified, "everything" will be build and release.
+#
+# Example:
+#   make release
+#   make release HELP=y
+#   make release WHAT=kubeedge/edgesite/keadm
+#   make release WHAT=kubeedge GOARM=GOARM7/GOARM8
```

Comment:
I think `\` will not be supported? If not supported, it's better not write in the help info :)
```
make release WHAT=kubeedge GOARM=GOARM7
make release WHAT=kubeedge GOARM=GOARM8
```